tests/virt_autotest/uefi_guest_verification.pm

```diff
@@ -34,22 +34,8 @@ sub run_test {
     # No machine type on aarch64 supports power management, or secure boot
     return $self if (is_aarch64);
 
-    if (is_kvm_host) {
-        if (is_sle) {
-            record_soft_failure("In order to implement pm features, current kvm virtual machine uses uefi firmware that does not support PXE/HTTP boot and secureboot. bsc#1182886 UEFI virtual machine boots with trouble");
-        }
-        elsif (is_alp) {
-            # The current default uefi firmware in alp kvm container supports secure boot,
-            # but does not support PXE/HTTP boot, and pm is not well supported either.
-            # Secure boot is supported with q35 machine type only
-            $self->check_guest_secure_boot($_) foreach (keys %virt_autotest::common::guests);
```

Comment:
This is the core part for secure boot checks -- check_guest_secure_boot. This is needed for sle16 -- this is also what we meant by enabling secure boot test for uefi guest.